src/compiler/crystal/macros/methods.cr

```diff
@@ -644,7 +644,13 @@ module Crystal
           StringLiteral.new(@value + piece)
         end
       when "camelcase"
-        interpret_argless_method(method, args) { StringLiteral.new(@value.camelcase) }
+        interpret_argless_method(method, args) do
+          lower = ((nargs = named_args) && (lower_arg = nargs["lower"]?)) ? lower_arg : BoolLiteral.new(false)
```

Comment:
Minor stylistic issue: I'd prefer proper `if` branches here instead of a convoluted conditional operator.